.github/workflows/demo-distribution.yml

```diff
@@ -1,4 +1,4 @@
-name: Firebase Test Builds
+name: Distribute Demo App
```

Comment:
Can we name this `Distribute Firebase Test Build`? We will later have a TestFlight version of this, too